### ISSUE TITLE
Repo cloner is not being able to pull Github repos

### ISSUE THREAD

**Issue by alpiquero** [konstellation-io/kdl-server#760]
When a Github repository is added as an external project Repo Cloner is being unable to clone it:
```
2022-03-16T11:56:43.076846486+01:00 2022-03-16T10:56:43.076717215Z INFO Found 1 new repositories to clone                                                                                                  
2022-03-16T11:56:43.079923796+01:00 2022-03-16T10:56:43.079829571Z INFO github.com is a known host                                                                                                         
2022-03-16T11:56:43.079957550+01:00 2022-03-16T10:56:43.079851039Z DEBUG Repository ny-house-price-estim with URL ssh://git@github.com/konstellation-io/ny-house-price-estimator.git found. Clone starting 
2022-03-16T11:56:44.202325914+01:00 2022-03-16T10:56:44.202195556Z ERROR Error cloning repository: unknown error: ERROR: You're using an RSA key with SHA-1, which is no longer allowed. Please use a newer client or a different key type. 
 ```
 
This is due the generated RSA keys are not being validated by Github sice they changed their acceptance key policy on March 15th:  https://github.blog/2021-09-01-improving-git-protocol-security-github/#when-are-these-changes-effective

**Comment by rrbarrero** [comment]
Although the update fixed the problem I reopen to investigate the origin of the problem.

Tests at 2396b14ba1e20338f0c4698edf67ecc9615249bb

- The error persist on version 5.4.2 of go-git 
- Although it doesn't work on the dashboard it's work from console on repoclone (same key)

**Comment by rrbarrero** [comment]
Fixed in e143d98ef6360e8170342af113ad75b95d7ec9ca updating go-git of repoCloner
